orca-test/src/main/groovy/com/netflix/spinnaker/orca/pipeline/ExecutionRunnerSpec.groovy

```diff
@@ -445,6 +446,41 @@ abstract class ExecutionRunnerSpec<R extends ExecutionRunner> extends Specificat
     contexts = [[region: "a"], [region: "b"]]
   }
 
+  def "does not consider before tasks of an internal parallel stage to complete the stage"() {
```

Comment:
similarly the stage isn't complete until the post tasks of a parallel stage complete?